docs/developers/tutorials/crowdfunding-p2.md

```diff
@@ -428,40 +421,34 @@ Contract functions can return in principle any number of results, that is why `"
 Finally, let's add the `claim` method. The `status` method we just implemented helps us keep the code tidy:
 
 ```rust
-#[endpoint]
-  fn claim(&self) -> SCResult<()> {
-      match self.status() {
-          Status::FundingPeriod => sc_error!("cannot claim before deadline"),
-          Status::Successful => {
-              let caller = self.blockchain().get_caller();
-              require!(
-                  caller == self.blockchain().get_owner_address(),
-                  "only owner can claim successful funding"
-              );
-
-              let sc_balance = self.get_current_funds();
-              self.send()
-                  .direct(&caller, &TokenIdentifier::egld(), 0, &sc_balance, b"claim");
-
-              Ok(())
-          },
-          Status::Failed => {
-              let caller = self.blockchain().get_caller();
-              let deposit = self.deposit(&caller).get();
-
-              if deposit > 0 {
-                  self.deposit(&caller).clear();
-                  self.send()
-                      .direct(&caller, &TokenIdentifier::egld(), 0, &deposit, b"claim");
-              }
-
-              Ok(())
-          },
-      }
-  }
+    #[endpoint]
+    fn claim(&self) {
+        match self.status() {
+            Status::FundingPeriod => sc_panic!("cannot claim before deadline"),
+            Status::Successful => {
+                let caller = self.blockchain().get_caller();
+                require!(
+                    caller == self.blockchain().get_owner_address(),
+                    "only owner can claim successful funding"
+                );
+
+                let sc_balance = self.get_current_funds();
+                self.send().direct_egld(&caller, &sc_balance);
+            },
+            Status::Failed => {
+                let caller = self.blockchain().get_caller();
+                let deposit = self.deposit(&caller).get();
+
+                if deposit > 0u32 {
+                    self.deposit(&caller).clear();
+                    self.send().direct_egld(&caller, &deposit);
+                }
+            },
+        }
+    }
 ```
 
-The only new function here is `self.send().direct()`, which simply forwards funds from the contract to the given address. The last argument is a message that gets saved on the blockchain too with the transaction.
+The only new function here is `self.send().direct_egld()`, which simply forwards EGLD from the contract to the given address. The last argument is a message that gets saved on the blockchain too with the transaction.
```

Comment:
"The last argument is a message that gets saved on the blockchain too with the transaction." message is no longer relevant.